docs/desktop/gnome/valuta.md

```diff
@@ -0,0 +1,50 @@
+﻿---
+ title: Valuta
+ author: Christine Belzie
+ contributors:
+---
+
+## Introduction
+
+ If you often travel or are moving abroad, ease your finanical planning with Valuta. This application quickly converts currencies between two countries.
+
+## Assumptions
+
+This guide assumes you have the following:
+
+- Rocky Linux
+- Flatpak
+- FlatHub
+
+## Installation Process
+
+1. Go to [Flathub.org](https://flathub.org), type "Valuta" in the search bar and click  **Install** ![Screenshot of the Valuta page on Flathub with the blue install button highlighted in a red square](images/01_valuta.png)
+ 
+  ![manual install script and run script](images/valuta-install.png)
+
+2. Copy the manual script in your terminal:
+
+```bash
+flapak install flathub io.github.idevecore.Valuta
+```
+
+3. Finally, copy the run command and run it in your terminal:
+
+```bash
+flatpak run io.github.idevecore.Valuta
+```
+
+## How to Use
+
+To use Valuta, do the following:
+
+1. Choose the country that you currently live in from the dropdown menu and type the amount of cash you want to spend.
+
+![Screenshot of Valuta app showing 1000 USD in the input field, with a grey arrow pointing down to a grey box showing 1000 USD](images/02_valuta.png)
+
```

Comment:
Since the screenshot comes after the first numbered list item, in order for the list to continue with item 2, rather than repeating item 1, it needs to be indented the 4 spaces. Also, while you are correct that the final screenshot (after number 2. here) can be left-justified, leave a blank line between it and the numbered list item.
```
1. Choose the country that you currently live in from the drop down menu and type the amount of cash you want to spend.

    ![Screenshot of Valuta app showing 1000 USD in the input field, with a grey arrow pointing down to a grey box showing 1000 USD](images/02_valuta.png) 

2. Choose the country you are traveling to....

![Screenshot of Valuta app showing 1000 USD in the input field, with a grey arrow pointing down to a grey box showing 1000 USD](images/02_valuta.png)

Reply:
So...I stick with one screenshot for both steps @sspencerwire?

Reply:
> So...I stick with one screenshot for both steps @sspencerwire?

No, my bad on that one. I just wasn't paying attention to the screenshot and had the other one in my clipboard, I was concerned just with the positioning. Last screenshot there should have been the one you have for that option. Sorry about that!